app/clients/youtube/video_statistics.rb

```diff
@@ -0,0 +1,24 @@
+module Youtube
+  class VideoStatistics < Client
+    def list(video_id)
+      path = "/videos"
+      query = {
+        part: "statistics",
+        id: video_id
+      }
```

Comment:
Little naming reco:

We want to retrieve the statistics to do so we are querying the `/videos` how about naming the `class Videos < Client` and making it more general purpose (ie not returning only the stats information)?


Also I know the Youtube api documentation talk about a "list" for getting a single object. I understand that you name this method accordingly. 
I would prefer a more restful naming such as
`retrieveStatistic` or `getStatistic`

A list I always expect it to return multiple objects

Reply:
Makes sense.